redhat-cop/pipeline-library
# Added exec plugin to execute check for missing tests to mvn

#### What is this PR About?
This PR is about where is the best place to execute the check for missing tests/docs. It feels better to do this via mvn, so anyone contributing see's this upfront, instead of raising a PR and travis failing.

#### How do we test this?
- Create a file in vars/ - i.e.: touch vars/new.groovy
- mvn clean install
- See missing bash execute and fail

cc: @redhat-cop/day-in-the-life
